File: pytools/examples/golang/src/client/proto/ack_role_rand_name_ok.go
package proto

import (
	"packet"
)

type AckRoleRandNameOk struct {
	uname                    string
}

func AckRoleRandNameOkDecode(pack *packet.Packet) *AckRoleRandNameOk {
	ackRoleRandNameOk := &AckRoleRandNameOk{}

	ackRoleRandNameOk.uname = pack.ReadString()
	return ackRoleRandNameOk
}

func (this *AckRoleRandNameOk) GetUname() string {
	return this.uname
}

File: pytools/examples/csharp/client/Protocol/AckRoleRandNameOk.cs
using System.Collections;
using System.Collections.Generic;


public class AckRoleRandNameOk
{
	private string _uname;


	public AckRoleRandNameOk(Packet packet)
	{
		this._uname = packet.ReadString();
	}


	public string uname
	{
		get { return this._uname; }
		set { this._uname = value; }
	}

}
